lib/sentry.ex

```diff
@@ -411,6 +412,7 @@ defmodule Sentry do
       |> CheckIn.new()
       |> Client.send_check_in(options)
     else
+      # add record dropped event here? Dropped due to....
```

Comment:
Ask on Discord because I don't think we need to report these here but maybe other SDKs do. Remove this comment in the meantime so that we can start shipping this PR.